4.2.5 release (#89)
blackbaud/skyux-colorpicker: CHANGELOG.md

# 4.2.5 (2021-04-30)

> - Fixed the colorpicker component to show previously selected color when clicking the cancel button. [#88]https://github.com/blackbaud/skyux-colorpicker/pull/88)

Reviewer: ```suggestion
- Fixed the colorpicker component to display the previously selected color after users select the cancel button. [#88](https://github.com/blackbaud/skyux-colorpicker/pull/88)
```
